src/Microsoft.ML.StandardLearners/Standard/SdcaRegression.cs

```diff
@@ -73,7 +73,7 @@ public Options()
         {
             Host.CheckNonEmpty(featureColumn, nameof(featureColumn));
             Host.CheckNonEmpty(labelColumn, nameof(labelColumn));
-            _loss = loss ?? Args.LossFunction.CreateComponent(env);
+            _loss = loss ?? SdcaTrainerOptions.LossFunction.CreateComponent(env);
```

Comment:
>SdcaTrainerOptions [](start = 28, length = 18)

Here as well? why not `Options`?

Reply:
Mostly for clarity, to help users distinguish between this property v/s class `Options`

---
In reply to: [258279715](https://github.com/dotnet/machinelearning/pull/2636#discussion_r258279715) [](ancestors = 258279715)